task/02-numbers-tasks.js

```diff
@@ -127,7 +131,11 @@ function getAngleBetweenVectors(x1, y1, x2, y2) {
  *     0     => 0
  */
 function getLastDigit(value) {
-    throw new Error('Not implemented');
+    let result = 0;
+    for(let i = 0; i < value.toString().length; i++){
+        result = value % 10;
+    }
+    return result;
```

Comment:
We can solve this task without loops.
(last digit of any decimal number is remainder of division by 10)